changelogs/current.yaml

```diff
@@ -98,5 +98,8 @@ new_features:
 - area: access_log
   change: |
     added support for number values in substitution format string in json_format.
+- area: lua
+  change: |
+    added stats for lua filter.
```

Comment:
please ref link to the new section you add in lua_filter.rst

Reply:
Done